FolfApp/src/components/courseDetails.re

```diff
@@ -0,0 +1,57 @@
+module CourseDetails = {
+  include ReactNative;
+  include ReactRe.Component.JsProps;
+  let name = "CourseDetails";
+  type par = Js.t {. red : Js.Null.t int, white : Js.Null.t int, blue : Js.Null.t int};
```

Comment:
Is it possible to say that two out of three can be null but at least one has to be and value ? 
Since if every one is null we should display something like "This course has not been logged yet"